Adds JSONVar for Nautobot Jobs (#4927)
nautobot/nautobot: nautobot/docs/development/jobs/index.md

Arbitrary text of any length. Renders as a multi-line text input field.

#### `JSONVar`

> Stores JSON, the provided JSON data must serializable. Renders as a multi-line text input field.

Reviewer: ```suggestion
Accepts JSON-formatted data of any length. Renders as a multi-line text input field.
```

Arbitrary text of any length. Renders as a multi-line text input field.

> #### `JSONVar`
> 
> Accepts JSON-formatted data of any length. Renders as a multi-line text input field. The variable passed to `run()` method on the job has been serialized to the appropriate Python objects.

Reviewer: ```suggestion
#### `JSONVar`

+++ 2.1.0

Accepts JSON-formatted data of any length. Renders as a multi-line text input field. The variable passed to `run()` method on the job has been serialized to the appropriate Python objects.
```
